src/color-slider/color-slider.js

```diff
@@ -63,7 +63,7 @@ const Self = class ColorSlider extends NudeElement {
 		if (["min", "max", "step", "value", "defaultValue"].includes(name)) {
 			prop.applyChange(this._el.slider, change);
 
-			let spinnerValue = this.tooltip === "progress" ? +(this.progress * 100).toPrecision(4) : this.value;
+			let spinnerValue = this.tooltip === "progress" ? +(this.progress * 100).toPrecision(4) : change.value;
```

Comment:
This will still fail if `this.tooltip` is `"progress"`. This entire code path is misguided, this line assumes you're propagating `value`, but you're actually propagating min, max, and step too.
I think we need to branch based on whether the property was min/max/step or value/defaultValue.

Reply:
Absolutely! I should trust my instincts more—I rejected my first fix in favor of this one (and made a mistake). 🤦‍♂️